sklearn/svm/tests/test_sparse.py

```diff
@@ -38,6 +39,7 @@
 iris.data = sparse.csr_matrix(iris.data)
 
 
+@ignore_warnings
```

Comment:
the warning is silenced, but the test will fail in 0.18, when default `decision_function_shape` will change to `'ovr'`.
I would rather add now `decision_function_shape='ovo'`.